maxim/tests/test_together.py

```
@@ -0,0 +1,552 @@
+                ],
+                max_tokens=200,
+                temperature=0.7
+            )
+            
+            # Verify response structure
+            self.assertIsNotNone(res)
+            self.assertTrue(hasattr(res, 'choices'))
+            self.assertTrue(len(res.choices) > 0)
+            self.assertIsNotNone(res.choices[0].message.content)
+            
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)